app/cafe.py

```diff
@@ -0,0 +1,19 @@
+from app.errors import (NotVaccinatedError,
+                        OutdatedVaccineError,
+                        NotWearingMaskError)
+from datetime import date

```

Comment:
Imports from standard library should come first, then third-party packages, and then local imports.

Reply:
Done! Thank you for your review